sources/javascripts/concat/base.js

```diff
@@ -537,6 +537,38 @@
     });
   };
 
+  // ===========================================================================
+  // Change product image position on narrower screens (mobile devices)
+  // ===========================================================================
+
+  var handleProductPageContent = function () {
+    $(document).ready(function () {
+      changeProductImagePos();
+    });
+
+    $(window).resize(debounce(function () {
+      changeProductImagePos();
+    }, 25));
+
+    var changeProductImagePos = function () {
+      var productGallery = $('.js-product-gallery');
+      var productImageContentBox = $('.js-content-item-box');
+      var buyBtnContent = $('.js-buy-btn-content');
+
+      if ($('.js-buy-btn-content .edy-buy-button-container').length >= 1) {
+        if ($(window).width() <= 640) {
```

Comment:
The width value should be dependant on the breakpoint of the design. The most accurate one here would be the [`$medium-down` breakpoint](https://github.com/Voog/design-santiago/blob/db264c000f039936c7e52357c1b5d4df9904bad8/sources/stylesheets/variables/_breakpoints.scss#L9) (as this is where the text goes under the product image.

Reply:
One note. Since in JS we haven't breakpoint constant we must use number `640` but `<`

```suggestion
        if ($(window).width() < 640) {
```